File: pulsar-third/pulsar-chrome-devtools/src/main/java/com/github/kklisura/cdt/protocol/events/webaudio/AudioParamCreated.java
package com.github.kklisura.cdt.protocol.events.webaudio;

/*-
 * #%L
 * cdt-java-client
 * %%
 * Copyright (C) 2018 - 2019 Kenan Klisura
 * %%
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *      http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 * #L%
 */

import com.github.kklisura.cdt.protocol.types.webaudio.AudioParam;

/** Notifies that a new AudioParam has been created. */
public class AudioParamCreated {

  private AudioParam param;

  public AudioParam getParam() {
    return param;
  }

  public void setParam(AudioParam param) {
    this.param = param;
  }
}

File: pulsar-core/pulsar-tools/pulsar-browser/src/main/kotlin/ai/platon/cdt/kt/protocol/events/webaudio/AudioParamCreated.kt
@file:Suppress("unused")
package ai.platon.cdt.kt.protocol.events.webaudio

import ai.platon.cdt.kt.protocol.types.webaudio.AudioParam
import com.fasterxml.jackson.`annotation`.JsonProperty

/**
 * Notifies that a new AudioParam has been created.
 */
data class AudioParamCreated(
  @param:JsonProperty("param")
  val `param`: AudioParam,
)
